labelbox/client.py

```diff
@@ -186,11 +186,27 @@ def check_errors(keywords, *path):
         if response_msg.startswith("You have exceeded"):
             raise labelbox.exceptions.ApiLimitError(response_msg)
 
-        prisma_error = check_errors(["INTERNAL_SERVER_ERROR"], "extensions",
-                                    "code")
-        if prisma_error:
-            raise labelbox.exceptions.InternalServerError(
-                prisma_error["message"])
+        resource_not_found_error = check_errors(["RESOURCE_NOT_FOUND"],
+                                                "extensions", "exception",
+                                                "code")
+        if resource_not_found_error is not None:
+            # Return None and let the caller methods raise an exception
+            # as they already know which resource type and ID was requested
+            return None
+
+        # A lot of different error situations are now labeled serverside
+        # as INTERNAL_SERVER_ERROR, when they are actually client errors.
+        # TODO: fix this in the server API
+        internal_server_error = check_errors(["INTERNAL_SERVER_ERROR"],
+                                             "extensions", "code")
+        if internal_server_error is not None:
+            message = internal_server_error.get("message")
+
+            if message.startswith("Syntax Error"):
```

Comment:
So the API is the reason why we check it this way right?

Reply:
Yep. For syntax errors the API now responds with "INTERNAL_SEVER_ERROR", but the message contains "Syntax Error". This is wrong, but yeah...